recipe/meta.yaml

```diff
@@ -39,53 +48,67 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python {{ python_min }}
     - setuptools >=65
     - setuptools-scm >=6.2
     - pip
   run:
-    - python >=3.9
-    - astropy
+    - python >={{ python_min }}
+    - astropy-base
     - boost-histogram
-    - ctapipe
+    - ctapipe-base
     - eventio
     - jsonschema
     - matplotlib-base
     - numpy
+    - particle
     - pymongo
     - pyproj
     - python-dotenv
     - pyyaml
+    - pytest
     - scipy
     - toml
+    - pip:
+       - ctao-dpps-cosmic-ray-spectra
```

Comment:
I don't think you are allowed to depend on pip packages

Reply:
I just learned this. So what can I do, as the ctao-dpps-cosmic-ray-spectra is not available through conda?

Reply:
I'll add it